Forge Networking Remastered Unity/Assets/Bearded Man Studios Inc/Scripts/NetworkManager.cs

```diff
@@ -667,5 +667,10 @@ public virtual void SceneReady(Scene scene, LoadSceneMode mode)
 
 			}
 		}
-	}
+
+        public static GameObject GetGameObjectByNetworkId(uint id)
```

Comment:
Does this need to be static since you are using `Instance` inside it anyway? Might be cleaner to remove the static keyword and use it through the instance.

Reply:
I found the static method easier & quicker than having to do `NetworkManager.Instance.GetGameObjectByNetworkId` as opposed to `NetworkManager.GetGameObjectByNetworkId`. 

Since it's a singleton, the static method gives the same results with the same consequences in terms of accessing the `NetworkManager`.